host/tests/rmfAudio_L3_TestCases/rmfAudio_test02_primaryJitterTest.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+import time
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path, "../"))
+
+from rmfAudio_L3_TestCases.rmfAudioHelperClass import rmfAudioHelperClass
+
+class rmfAudio_test02_primaryJitterTest(rmfAudioHelperClass):
+    """
+    Test class to open capture Primary Audio Data and check if there is jitter detected.
+
+    This class uses the `rmfAudioClass` to run primary data capture,
+    downloading necessary test assets, setting up RMF Audio Capture
+    and performing verification of audio capture rate.
+    """
+    def __init__(self):
+        """
+        Initializes the test class with test name, setup configuration, and sessions for the device.
+
+        Args:
+            None
+        """
+        self.testName  = "test02_primaryJitterTest"
+        super().__init__(self.testName, '2')
+
+    def testFunction(self):
+        """
+        The main test function that verifies primary data capture with input audio.
+
+        This function:
+        - Starts Primary Data capture.
+        - Tracks bytes received for Audio data on primary interface.
+        - Monitors bytes received in regular intervals
+
+        Returns:
+            bool: Final result of the test.
+        """
+
+        self.log.testStart(self.testName, '2')
+        result = True

```

Comment:
Initially result should be False correct?

Reply:
Updated this

Reply:
could you please update on this comment?